simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,18 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(lst: List[int]) -> str:
+    for num in lst:
+        if num != 0 and num != 1:
+            return -1
+    
+    binaryStr = ""
+    for i in range(len(lst)):
```

Comment:
range(len(lst)) here are find. You might or might not fix it. But I recommend having a "lenLst = len(lst)" in the beginning and using it here. Up to you :)